element_lab/lab.py

```diff
@@ -77,18 +77,52 @@ class Protocol(dj.Lookup):
     protocol                : varchar(16)
     ---
     -> ProtocolType
-    protocol_description=''        : varchar(255)
+    protocol_description='' : varchar(255)
     """
 
 
 @schema
 class Project(dj.Lookup):
     definition = """
+    -> Lab
     project                 : varchar(32)
     ---
-    project_description=''         : varchar(1024)
+    project_description=''  : varchar(1024)
+    # Below included for archival export (e.g., NWB)
+    repositoryurl=''        : varchar(256) # URL to code for replication
+    repositoryname=''       : varchar(32)  # name of repository
+    pharmacology = ''       : varchar(2048) # Drugs used, how/when administered
+    viruses=''              : varchar(2048) # ID, source, date made, injection loc, volume
+    slices=''               : varchar(2048) # If slicing, preparation thickness, orientation, temperature, and bath solution
+    stimulus=''             : varchar(2048) # Generation method, how/when/where presented
+    surgery=:''             : varchar(2048) # Description of surger(y/ies), who performed, when relative to other events
```

Comment:
Many of these attributes here are redundant, we already have them in other tables in this element-lab, you can retrieve them by joining other tables together.
This is a bit forcing this table to accommodate NWB attributes, e.g. `slices`, `stimulus`, `surgery`, etc. wouldn't make sense to be an attribute of the `Project` table.

Reply:
I'm sorry - It's not clear to me which are already included elsewhere in `element-lab`. Were you thinking they would be covered by `project_description`? 

My goal here was to allow the user to maximally specify if they chose. The [documentation I was reading](https://nwb-schema.readthedocs.io/en/latest/format.html#groups-general) implied free response text describing the nature of the experiment broadly. I mapped this on to the project level. Should we instead optionally gather this information at `session.SessionNote`?

Reply:
Perhaps we should move the `pharmacology`, `viruses`, `slices`, and `surgery` attributes to the `lab.Protocol` or `session.SessionNote` table?

Reply:
Virus information could be stored in a separate schema altogether.  See `atlab/commons` [virus schema](https://github.com/atlab/commons/blob/master/python/commons/virus.py).